decidim-admin/app/helpers/decidim/admin/application_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+module Decidim
+  module Admin
+    # Custom helpers, scoped to the admin panel.
+    #
+    module ApplicationHelper
+      def title
+        "Decidim"
+      end
+
+      def field_name(model, field)
```

Comment:
Remove this as it's not used anymore (actually, it should be gone un `decidim-system` as well).